c_common.c

```diff
@@ -81,6 +81,9 @@ unsigned char *parse_received_message(unsigned char *received_buf,
                                       unsigned char *message_type,
                                       unsigned int *data_buf_length) {
     *message_type = received_buf[0];
+    if (*message_type == AUTH_ALERT) {
+        return received_buf + 1;
+    }
```

Comment:
You claimed `c_common.c` only contains reformatting. However, this change does not look like just reformatting. Please either update the PR description or change the code here.

Reply:
I updated the PR description. Thank you for having a look.